fendermint/vm/topdown/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+mod cache;
+
+use ipc_sdk::cross::CrossMsg;
+use ipc_sdk::ValidatorSet;
+use serde::{Deserialize, Serialize};
+
+type BlockHeight = u64;
+
+/// The finality proof for IPC parent at certain height.
```

Comment:
```suggestion
/// The finality view for IPC parent at certain height.
```
I don't know what's the best phrase but I always find it strange to read that this structure _proves_ anything.